support/cas-server-support-pac4j-core-clients/build.gradle

```diff
@@ -3,7 +3,11 @@ dependencies {
     implementation project(":core:cas-server-core-tickets-api")
     implementation project(":core:cas-server-core-web-api")
     implementation project(":core:cas-server-core-authentication-api")
+    implementation project(":core:cas-server-core-util")
+    implementation project(":core:cas-server-core-configuration")
```

Comment:
You will need to pull in the configuration-api

Reply:
Changed to cas-server-core-configuration-api, thanks!